### PR TITLE
update sizes.txt

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: data/cav5_comb4/3D/scalability/sizes.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
...
-21.5  96k
+| Mesh size parameter | Tets (approx.) |
+|---------------------|----------------|
+| 48                  | 23810          |
+| 30.5                | 47675          |
+| 21.5                | 95922          |
+| 15                  | 227891         |
+| 10                  | 672239         |
+| 6.4                 | 2376197        |
+| 5                   | 4849277        |
```
Comment (MTCam) [inline data/cav5_comb4/3D/scalability/sizes.txt:9]: I like this, but we need to sync it up with mine.

Comment (MTCam) [inline data/cav5_comb4/3D/scalability/sizes.txt:9]:
```suggestion
| Mesh size parameter | Target | Tets (approx.) |
|---------------------|----------------|--------|
| 48           | 24k       |      23810        |
| 48           | 24k       |      24036        | 
| 30.5        | 48k       |      47908        |
| 21.5         | 96k       |      96425        |
| 16.05       | 192k     |     192658        |
| 12.3          | 384k    |     383014       |
| 9.535       | 768k     |     768861       |
| 7.45          | 1536k   |     1536571      |  
| 5.8545     | 3072k   |     3071835     | 
| 4.61          | 6144k   |     6142150      | 
| 3.6389     | 12288k |     12283032   |
```

Comment (MTCam) [reply data/cav5_comb4/3D/scalability/sizes.txt:9]: Thanks for your patience.

File: data/cav5_comb4/3D/scalability/sizes.txt [hunk 2]
```diff
@@ -1,3 +1,12 @@
-48    24k
-30.5  48k
-21.5  96k
+| Mesh size parameter | Target  | Tets (approx.) |
```
[the remark this answers is not in the document]
Comment (matthiasdiener) [reply data/cav5_comb4/3D/scalability/sizes.txt:1]: Ok, but what is the difference between Target and Num Elements (considering that the actual number will vary anyways)?

**Review by anderson2981 — APPROVED** [review]

File: data/cav5_comb4/3D/scalability/sizes.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (MTCam) [reply data/cav5_comb4/3D/scalability/sizes.txt:1]: Target: this is what you want
Num Elements: This is what you ~~get~~ got

[the remark this answers is not in the document]
Comment (matthiasdiener) [reply data/cav5_comb4/3D/scalability/sizes.txt:1]: But Num Elements isn't what you get, since every time this gets rerun the number of tets changes...

**Pull request merged**